ProjectLibertyLabs/gateway
# feat: add docker-compose configurations for Developer track

# Problem

In order to support the Developer Track, `start.sh` needs to be able to run the gateway services using the published containers. 

See: #630 

# Solution

- `start.sh` refactored to implement a new question: whether or not to use published containers, and to allow selectively starting gateway services
- `stop.sh` updated to match `start.sh` refactoring
- `test_start.bats` added to use the bats-core framework for bash script testing
- `docker-compose-published.yaml` added to support using the published containers. Note: using docker compose overrides was not workable as overrides were not able to remove dependencies from the base docker compose.
- 

## Steps to Verify:

1. Execute `start.sh` 
2. Answer the questions to select published containers.
3. Verify that docker starts the published containers and does not build from scratch.
